src/main/java/edu/kis/powp/command/factory/Joe1Factory.java

```diff
@@ -0,0 +1,30 @@
+package edu.kis.powp.command.factory;
+
+import edu.kis.powp.command.ComplexCommand;
+import edu.kis.powp.command.OperateToCommand;
+import edu.kis.powp.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class Joe1Factory {
```

Comment:
It is not a command factory. It doesn't enable clients to obtain a command instance.